UnitsNet/UnitDefinitions/TemperatureDelta.json

```diff
@@ -3,9 +3,106 @@
   "BaseUnit": "KelvinDelta",
   "XmlDoc": "Difference between two temperatures. The conversions are different than for Temperature.",
   "Units": [
+    {
+      "SingularName": "Kelvin",
+      "PluralName": "Kelvins",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "▲K" ]
```

Comment:
Change symbols to ∆

See discussions:
https://github.com/angularsen/UnitsNet/issues/327
https://github.com/angularsen/UnitsNet/issues/328